lib/AssetGraphPlugin.js

```diff
@@ -12,8 +12,12 @@ const path = require('path');
  */
 class AssetGraphPlugin {
   constructor(target) {
-    // get output path, relative from webpack.config.js file
-    this.output = path.join(process.cwd(), target);
+	if (path.isAbsolute(target)) {
```

Comment:
We can simply use path.resolve here, it will prioritize absolute path

```
this.output path.resolve(process.cwd(), target)
```

if target is absolute path, it will return that, if not, it will return similar to `path.join` so no need for if `path.isAbsolute`